### DISCUSSION TITLE
Searching files and text pattern in archives recursively

### DISCUSSION THREAD

**Discussion by corebuzo** [doublecmd/doublecmd#1348] [Suggestions]
Maybe an improved searching in archives can implemented ?

i.e. Readme.zip
         |---Readme.txt
         |--- Readme1.zip
         ___|---Readme.txt
         ___| ...
         |--- Readme2.zip
         ___|---Readme.txt
         ___|---...
         ___|---Third.zip
         ______|---Readme.txt

All Readme's contains the same content. Searching for the filename will show results in all files. 
Searching for an containing text will just display files from Readme.zip ?

Search unlimited subdirectories implies all archives will be scanned recursively, therefore all Readme.txt containing the search pattern will be displayed ? Searching for a file Mask works intuitively, but files in Third.zip wont displayed.

P.S.: Deleting a zip out of zip - try it !
[README.ZIP](https://github.com/doublecmd/doublecmd/files/10106618/README.ZIP)

**Comment by avmaksimov** [comment]
Did you try to use another plugin for it?
